docs/DUAL_BAND.md

```diff
@@ -0,0 +1,19 @@
+# mLRS Documentation: Dual Band #
+
+([back to main page](../README.md))
+
+**Important: Dual band functionality is experimental and currently only available in the branch 'dev-dualband2'. Firmware binaries need to be compiled manually for the specific hardware.**
+
+mLRS allows for a dual-band setup, where the Tx module and receiver both transmit and receive in the 2.4 GHz and 868/915 MHz bands (also the combination 2.4 GHz and 433 MHz/70 cm is possible). 
```

Comment:
Small change:

'mLRS allows for a dual-band setup, where the Tx module and receiver both transmit and receive in the 2.4 GHz and 868/915 MHz (or 433 MHz/70 cm) bands simultaneously.'

Reply:
great. thx. done.